test/simplifyExpression/distributeSearch/distributeSearch.test.js

```diff
@@ -50,3 +50,24 @@ describe('distribute', function () {
   ];
   tests.forEach(t => testDistributeSteps(t[0], t[1]));
 });
+
+describe('distribute with fractions', function () {
```

Comment:
could you also add a section for testing with substeps? because the code you changed does a specific step that's only part of the overall distribution (right? cause for example in your first test it then goes on to multiply x^2 by x^3 and stuff)